src/Gaufrette/Adapter/Local.php

```diff
@@ -250,17 +249,13 @@ protected function ensureDirectoryExists($directory, $create = false)
      *
      * @param string $directory Path of the directory to create
      *
-     * @throws InvalidArgumentException if the directory already exists
-     * @throws RuntimeException         if the directory could not be created
+     * @throws \InvalidArgumentException if the directory already exists
```

Comment:
@AntoineLelaisant Why? Even if the error is silenced, if `mkdir` fails and the directory does not exists, it will throw the exception.